addons/binding/pom.xml

```diff
@@ -38,6 +38,7 @@
     <module>org.openhab.binding.folding</module>
     <module>org.openhab.binding.freebox</module>
     <module>org.openhab.binding.fronius</module>
+    <module>org.openhab.binding.ftpupload</module>
```

Comment:
Please also add the binding to https://github.com/openhab/openhab2-addons/blob/master/features/openhab-addons/src/main/feature/feature.xml, otherwise it won't turn up in the distro.